tools/benchmark/cmd/watch.go

```diff
@@ -209,7 +209,7 @@ func benchPutWatches(clients []*clientv3.Client, wk *watchedKeys) {
 		}
 	}()
 
-	limit := rate.NewLimiter(rate.Limit(watchPutRate), 1)
+	limit := rate.NewLimiter(rate.Limit(watchPutRate), watchPutTotal)
```

Comment:
I assume that the intent is to have the rate of `put's` being effectively unlimitted. 

Can we use `rate.Inf` constant (https://pkg.go.dev/golang.org/x/time/rate#pkg-constants) that seems to be dedicated for such scenario. Then watchPutTotal will not have really any meaning: 

```
func NewLimiter(r [Limit](https://pkg.go.dev/golang.org/x/time/rate#Limit), b [int](https://pkg.go.dev/builtin#int)) *[Limiter](https://pkg.go.dev/golang.org/x/time/rate#Limiter)

As a special case, if r == Inf (the infinite rate), b is ignored.
```

so the code would be like: 

```
watchPutLimit:=rate.Inf
if watchPutRate>0 {
  watchPutLimit=rate.Limit(watchPutRate);
}
 
limit := rate.NewLimiter(watchPutLimit, watchPutTotal)
```

Reply:
Thanks your suggestion.